### PR TITLE
Update live-response-command-examples.md

### PR DESCRIPTION
**Pull request by arnydo**
Add comment regarding the \\ indicating a registry value vs key.
Added example for remediating a registry value.

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/live-response-command-examples.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ registry HKEY_CURRENT_USER\Console
 ```
 
 ```console
-# Show information about a specific registry value
+# Show information about a specific registry value (the double backslash \\ indicates a registry value vs key)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/live-response-command-examples.md]:
```suggestion
# Show information about a specific registry value (the double backslash \\ indicates a registry value versus key)
```

File: microsoft-365/security/defender-endpoint/live-response-command-examples.md [hunk 2]
```diff
@@ -192,6 +192,11 @@ remediate file c:\Users\user\Desktop\malware.exe
 remediate process 7960
 ```
 
+```console
+# Remediate a registry value (the double backslash \\ indicates a registry value vs key)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/live-response-command-examples.md]:
```suggestion
# Remediate a registry value (the double backslash \\ indicates a registry value versus key)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arnydo, kindly commit the minor suggested changes. Many thanks.

**Comment by arnydo** [comment]
I have committed the suggested changes.
Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by arnydo** [comment]
Thank you for the merge!

**Pull request merged**